docs/shared/migration/adding-to-existing-project.md

```diff
@@ -322,21 +302,30 @@ To connect to Nx Cloud:
 - Commit and push your changes
 - Go to [https://cloud.nx.app](https://cloud.nx.app), create an account, and connect your repository
 
-#### Connect to Nx Cloud Manually
-
-If you are not able to connect via the automated process at [https://cloud.nx.app](https://cloud.nx.app), you can connect your workspace manually by running:
-
 ```shell
 npx nx connect
 ```
 
-You will then need to merge your changes and connect to your workspace on [https://cloud.nx.app](https://cloud.nx.app).
+When you are prompted to choose your CI provider, choose `GitHub Actions`.
```

Comment:
There's no prompt that I see. The workflow file will be added in next section.

(same for other standalone tutorials)